docs/whatsnew/0.8.rst

```diff
@@ -0,0 +1,140 @@
+.. doctest-skip-all
+
+.. _whatsnew-0.8:
+
+**************************
+What's New in SunPy 0.8?
+**************************
```

Comment:
we have a script for that in case you're interested:

https://github.com/astropy/astropy-tools/blob/master/unify_section_headings.py